tasks.py

```diff
@@ -205,5 +208,21 @@ def __init__(self, *args, **kwargs):
     server.serve_forever()
 
 
+# -----------------------------------------------------------------------------
+@task
+def web_build(ctx):
+    # Step 1: build the wheel
+    build(ctx)
+    # Step 2: Copy the wheel to the web folder, so the http server can access it
+    newest_wheel = Path(max(glob.glob('dist/*.whl'), key=lambda f: os.path.getmtime(f)))
+    shutil.copy(newest_wheel, Path('web/'))
+    # Step 3: Write wheel's name to web/packageFile
+    with open(Path('web', 'packageFile'), mode='w') as package_file:
```

Comment:
It might be simpler to just copy the built `.whl` file to just `web/bumble.whl` rather than go through the indirection of a text file pointing to the built file.
Then the web page can just reference `?package=bumble.whl`.

Reply:
Unfortunately doing that results in this error message:
```
packaging.utils.InvalidWheelFilename: Invalid wheel filename (wrong number of parts): bumble
```

Reply:
Bummer.